runtime/benches/bloom.rs

```diff
@@ -97,3 +98,48 @@ fn bench_sigs_hashmap(bencher: &mut Bencher) {
     });
     assert_eq!(falses, 0);
 }
+
+#[bench]
+fn bench_add_hash(bencher: &mut Bencher) {
+    let mut rng = rand::thread_rng();
+    let hash_values: Vec<_> = std::iter::repeat_with(|| Hash::new_rand(&mut rng))
+        .take(1200)
+        .collect();
+    let mut fail = 0;
+    bencher.iter(|| {
+        let mut bloom = Bloom::random(1287, 0.1, 7424);
+        for hash_value in &hash_values {
+            bloom.add(hash_value);
+        }
+        let index = rng.gen_range(0, hash_values.len());
+        if !bloom.contains(&hash_values[index]) {
+            fail += 1;
+        }
+    });
+    assert_eq!(fail, 0);
+}
+
+#[bench]
+fn bench_add_hash_atomic(bencher: &mut Bencher) {
+    let mut rng = rand::thread_rng();
+    let hash_values: Vec<_> = std::iter::repeat_with(|| Hash::new_rand(&mut rng))
+        .take(1200)
+        .collect();
+    let mut fail = 0;
+    bencher.iter(|| {
+        let bloom: AtomicBloom<_> = Bloom::random(1287, 0.1, 7424).into();
+        // Intentionally not using parallelism here, so that this and above
+        // benchmark only compare the bit-vector ops.
+        // For benchmarking the parallel code, change bellow for loop to:
```

Comment:
curious, how did this compare to the above `bench_add_hash`?

Reply:
Even without parallism, the atomic one is faster:
```
test bench_add_hash        ... bench:     153,569 ns/iter (+/- 6,109)
test bench_add_hash_atomic ... bench:     135,103 ns/iter (+/- 6,918)
```

With `par_iter`:
```
test bench_add_hash        ... bench:     151,852 ns/iter (+/- 9,219)
test bench_add_hash_atomic ... bench:      65,663 ns/iter (+/- 4,359)
```